Tests/SonarScanner.MSBuild.TFS.Test/Classic/BinaryToXmlCoverageReportConverterTests.cs

```diff
@@ -81,52 +82,17 @@ public void Conv_OutputIsCaptured()
             var inputFilePath = Path.Combine(testDir, "input.txt");
             File.WriteAllText(inputFilePath, "dummy input file");
 
-            var converterFilePath = Path.Combine(testDir, "converter.bat");
-            File.WriteAllText(converterFilePath,
-@"
-echo Normal output...
-echo Error output...>&2
-echo Create a new file using the output parameter
-echo foo > """ + outputFilePath + @"""");
-
             // Act
-            var success = BinaryToXmlCoverageReportConverter.ConvertBinaryToXml(converterFilePath, inputFilePath, outputFilePath, logger);
+            var success = BinaryToXmlCoverageReportConverter.ConvertBinaryToXml(inputFilePath, outputFilePath, logger);
 
             // Assert
-            success.Should().BeTrue("Expecting the process to succeed");
+            success.Should().BeFalse();
 
-            File.Exists(outputFilePath).Should().BeTrue("Expecting the output file to exist");
-            TestContext.AddResultFile(outputFilePath);
+            File.Exists(outputFilePath).Should().BeFalse("Conversion failed");
 
-            logger.AssertInfoLogged("Normal output...");
-            logger.AssertErrorLogged("Error output...");
-        }
-
-        [TestMethod]
-        [WorkItem(72)] // Regression test for bug #72: fail the conversion if the output file is not created
-        public void Conv_FailsIfFileNotFound()
-        {
-            // Arrange
-            var logger = new TestLogger();
-            var testDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
-
-            var outputFilePath = Path.Combine(testDir, "output.txt");
-
-            var inputFilePath = Path.Combine(testDir, "input.txt");
-            File.WriteAllText(inputFilePath, "dummy input file");
-
-            var converterFilePath = Path.Combine(testDir, "converter.bat");
-            File.WriteAllText(converterFilePath, @"REM Do nothing - don't create a file");
-
-            // Act
-            var success = BinaryToXmlCoverageReportConverter.ConvertBinaryToXml(converterFilePath, inputFilePath, outputFilePath, logger);
-
-            // Assert
-            success.Should().BeFalse("Expecting the process to fail");
-            logger.AssertErrorsLogged();
-            logger.AssertSingleErrorExists(outputFilePath); // error message should refer to the output file
-
-            File.Exists(outputFilePath).Should().BeFalse("Not expecting the output file to exist");
+            logger.AssertErrorLogged(@$"Failed to convert the downloaded code coverage tool to XML. No code coverage information will be uploaded to SonarQube.
+Check that the downloaded code coverage file ({inputFilePath}) is valid by opening it in Visual Studio. If it is not, check that the internet security settings on the build machine allow files to be downloaded from the Team Foundation Server machine.");
```

Comment:
Aren't the coverage files already on the machine? I don't understand why we need to mention "downloaded"

Reply:
The old TFS solution downloaded the coverage file from TFS. This may still be in use today.